src/PennyBlackPlugin.php

```diff
@@ -43,4 +45,13 @@ public static function createSettingsLink($links)
 
         return $links;
     }
+
+    public static function getVersion(): string
+    {
+        if (!isset(self::$VERSION)) {
+            $pluginData = get_file_data(__DIR__ . '/../woocommerce-pennyblack.php', array('Version' => 'Version'), false);
+            self::$VERSION = $pluginData['Version'];
```

Comment:
this is not wonderful using statics, but the alternative was passing version numbers down through lots of classes or a major restructure which was not appealing.